_security-plugin/configuration/security-admin.md

```diff
@@ -1,11 +1,14 @@
 ---
 layout: default
-title: Apply changes with securityadmin.sh
+title: Apply changes with the securityadmin script
 parent: Configuration
 nav_order: 20
 ---
 
-# Apply changes using securityadmin.sh
+# Apply changes using the securityadmin script
```

Comment:
Should the heading and the title match (with vs. using)?

Reply:
In this situation, it makes perfect sense, so I'll change it. Sometimes though we have to keep them different.